nextcord/application_command.py

```diff
@@ -536,6 +548,27 @@ def cog_application_command_check(self, interaction: Interaction) -> bool:
         """
         return True
 
+    @_cog_special_method
+    async def cog_application_command_error(
+        self, interaction: Interaction, error: ApplicationError
+    ) -> None:
+        """A special method that is called whenever an error
+        is dispatched inside this cog.
```

Comment:
```suggestion
        """A special method that is called whenever an error is dispatched inside this cog.
```
Summaries should be on the first line.